src/lib/format/l20n/ast/parser.js

```diff
@@ -485,7 +487,7 @@ export default {
 
     const junk = new AST.JunkEntry(
       this._source.slice(this._curEntryStart, nextEntry));
-    if (this._config.pos) {
+    if (AST.Node._config.pos) {
```

Comment:
I think I prefer the pos state to be on the parser state, and not on AST.Node.

Note, we'd expose that config on each node we return from the parse cycle, right?